test-harness/tests/conftest.py

```diff
@@ -246,9 +246,20 @@ def f(uid, *args, **kwargs):
 
 
 @pytest.fixture(scope='session')
+def superuser_user_header(valid_jwt_generator):
+    """This fixture further simplifies JWT handling by providing a ready-to-use
+    headers with a valid  superuser JSON Web Token for `requests` module to
+    use"""
+    token = valid_jwt_generator(uid='root')
```

Comment:
Where does the relation between superuser and `root` come from? That concept is not known in DC/OS. Is that a relation built into the test harness/mock system?